packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -55,7 +55,12 @@ export class Popup extends BaseComponent<IPopupProps, {}> {
   }
 
   public render() {
-    let { role, className, ariaLabel, ariaLabelledBy, ariaDescribedBy } = this.props;
+    let { role, className, ariaLabel, ariaLabelledBy, ariaDescribedBy, style } = this.props;
+
+    let needsVerticalScrollBar = false;
+    if (this.refs.root) {
```

Comment:
Right now this is leveraging the fact that popup is rendered by a callout using multiple passes, so we'd have a valid root on the second render. This is the same thing that calloutcontent uses for positioning and it gives us the perf we want by not introducing a reflow or another render (like we would get if we did this in componentdidupdate),  but it does feel a bit weird.